### PR TITLE
Fixes planet icon smoothing

### PR DESCRIPTION
**Pull request by Qustinnus**
:cl: Floyd / Qustinnus
fix: Fixes planet icon smoothing
fix: Fixes asteroid sand in mountains (partially)
tweak: lavaland mobs now spawn in ruins. rip you guys
/:cl:

ok ike is a dildo and his to_chat wrapper affected a bitshift.

besides that I dug into how mineral turf works and I made some subtypes of some turfs which means we can now use them properly for planets. 

Only bugs I still need to fix is that

1. when you dig / explode the ground, it goes black due to missing the icon state (grass_dug for example). Sprite issue

2. when you dig a mineral itll leave behind asteroid sand, fix for this is making subtypes of those mineral turfs but im too tired rn

planets no longer look like shit now.

### CODE REVIEW

File: code/datums/stars.dm [hunk 1]
```diff
@@ -172,13 +172,13 @@
 				nav_icon_name = "icy"
 				icon_layers += "p_icy"
 
-			if(101 to 130)
+			if(51 to 100)
```
Comment (ninjanomnom) [inline code/datums/stars.dm]: The random number goes up to 130 not 100. Also this sneakily changes planet ratios in the galaxy to massively increase habitable planets spawning.

Ratio before:
5:5:3

Ratio after
1:1:2

Comment (Qustinnus) [inline code/datums/stars.dm]: tweak: raises exo planet spawns, lowers ice and lava planet spawns

>sneakily

File: code/game/turfs/simulated/floor/plating/asteroid.dm [hunk 2]
```diff
@@ -188,6 +188,18 @@
 /turf/open/floor/plating/asteroid/airless/cave/volcanic/has_data //subtype for producing a tunnel with given data
 	has_data = TRUE
 
+/turf/open/floor/plating/asteroid/airless/cave/planet
+	mob_spawn_list = list(/mob/living/simple_animal/hostile/asteroid/goliath/beast = 10, /mob/living/simple_animal/hostile/spawner/lavaland/goliath = 3, \
+		/mob/living/simple_animal/hostile/asteroid/basilisk/watcher = 8, /mob/living/simple_animal/hostile/spawner/lavaland = 2, \
```
Comment (Qustinnus) [inline code/game/turfs/simulated/floor/plating/asteroid.dm]: you changed the spawnrate of mobs in asteroid turf fuckign calling the police on u make a new pr u cunt

**Comment by Qustinnus** [comment]
im tired

**Comment by Qustinnus** [comment]
pls just merge this

**Comment by ike709** [comment]
nah the real dildo is whoever made the to_chat regex

**Pull request merged**